### PR TITLE
Gulp - clean and add "disc" analysis task

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

**Review by danfinlay — APPROVED** [review]
Oh nice, I like `disc`, fun to have it as a task!

File: gulpfile.js [hunk 1]
```diff
@@ -243,21 +237,65 @@ function copyTask(opts){
...
   var browserifyOpts = assign({}, watchify.args, {
     entries: ['./app/scripts/'+opts.filename],
-    debug: true,
     plugin: 'browserify-derequire',
+    debug: debug,
+    fullPaths: debug,
   })
 
-  var bundler = browserify(browserifyOpts)
-  bundler.transform('brfs')
```
Comment (flyswatter) [inline gulpfile.js]: We actually rely on `brfs` for our CI test suite.

**Review by danfinlay — CHANGES_REQUESTED** [review]
Found the bug killing our CI tests.

**Review by danfinlay — APPROVED** [review]

**Comment by flyswatter** [comment]
Oooh, if putting `brfs` in the `package.json` transforms instead of the gulpfile works, that's much cleaner!

**Pull request merged**